### DISCUSSION TITLE
Moviment or person detect with a sound alarm?

### DISCUSSION THREAD

**Discussion by Pedroov** [roflcoopter/viseron#662] [Q&A] [answered]
So im about to test the Viseron, just waiting for my ip camera delivery. I would like to know if the application has a sound or any kind alarm function when the camera detects movement or people through some of the add-ons. If not, it would be a pretty cool addition to make.

**Comment by roflcoopter** [comment, marked as answer]
It is not possible to to this directly in Viseron, you would need some sort of automation system, like Home Assistant to do this.
If your cameras have sirens it could be possible to trigger those in the future, but creating camera specific implementations is hard since it means i need to get hold of a camera during development
